packages/grid/_modules_/grid/hooks/features/columnMenu/useGridColumnMenu.ts

```diff
@@ -77,4 +63,5 @@ export const useGridColumnMenu = (apiRef: GridApiRef): void => {
   );
 
   useGridApiEventHandler(apiRef, GridEvents.columnResizeStart, handleColumnResizeStart);
+  useGridApiEventHandler(apiRef, GridEvents.rowsScroll, handleRowsScroll);
```

Comment:
Would it make sense to simplify even futher?
```suggestion
  useGridApiEventHandler(apiRef, GridEvents.columnResizeStart, hideColumnMenu);
  useGridApiEventHandler(apiRef, GridEvents.rowsScroll, hideColumnMenu);
```